public/js/favorite.js

```diff
@@ -0,0 +1,54 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Can you please fix the indentation of this file? It uses both tabs and spaces and completely breaks on github.